libretro-common/cdrom/cdrom.c

```diff
@@ -442,7 +443,7 @@ static int cdrom_send_command(libretro_vfs_implementation_file *stream, CDROM_CM
 
          lba_req = cdrom_msf_to_lba(cmd[3], cmd[4], cmd[5]);
 
-         if (stream->cdrom.last_frame_valid && lba_req == stream->cdrom.last_frame_lba)
+         if (lba_req == stream->track->last_frame_lba)
```

Comment:
`last_frame_valid` was not necessary. `last_frame_lba` is set to an invalid value if the buffered contents are invalidated.